### DISCUSSION TITLE
outreachy

### DISCUSSION THREAD

**Discussion by mildrette** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#24] [Outreachy 2022] [answered]
Hello! Mildred fonka outreachy Intern, I am currently working on creating a subPage in the OSCSA website under the topic Open HardWare and I am having some issues i will love clearifications on. 
Am I to write a whole site on open hardware or i am to add refers links to other webpages talking about open Hardware??
please  help??

**Comment by BatoolMM** [comment, marked as answer]
Welcome @mildrette, thank you for your work!

We already have. a PR opened for open hardware ##36, we'll be adding more tasks today. Looking forward to your contributions!

**Reply by mildrette** [reply]
okay. i will be looking forward to it
